.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -55,6 +55,8 @@ assignees: ''
 - [ ] Run sanity check of Helm install using connectivity-check script.
       Suggested approach: Follow the full [GKE getting started guide].
 - [ ] Check draft release from [releases] page and publish the release
+- [ ] Check if all docker images are available before announcing the release
+      `make -C install/kubernetes/ check-docker-images`
```

Comment:
Presumably this should be part of the `Wait for docker builds to complete` item above?

Reply:
Good point!